### PR TITLE
Dont forget the submodules

### PR DESCRIPTION
**Pull request by brenthuisman**

### CODE REVIEW

File: example/izhikevich2007_network.nml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
```
Comment (thorstenhater) [inline example/izhikevich2007_network.nml:1]: Sadly, Arbor has no support for Izhikevich type neurons, a different example with either IAF or Cable types would be better.

**Review by thorstenhater — CHANGES_REQUESTED** [review]

File: example/izhikevich2007_network.nml (hunk 1 above)
Comment (brenthuisman) [reply example/izhikevich2007_network.nml:1]: Can we keep it anyway? It's a concise demo of a network, and it has it's own tutorial on the nml docs: https://docs.neuroml.org/Userdocs/IzhikevichNetworkExample.html

Comment (thorstenhater) [reply example/izhikevich2007_network.nml:1]: Sure, but it won't run in Arbor as of now.
You could fudge it and replace Iz -> IaF with the proper params.

Comment (brenthuisman) [reply example/izhikevich2007_network.nml]: I'll move it to the network branch anyway.

**Comment by thorstenhater** [comment]
:wave: I am closing this, it's changes were eaten up by other PRs. Feel free to re-open if needed.

**Pull request closed without merging**